MoreLinq/Repeat.cs

```diff
@@ -37,14 +37,28 @@ public static IEnumerable<T> Repeat<T>(this IEnumerable<T> sequence, int count)
             return RepeatImpl(sequence, count);
         }
 
-        private static IEnumerable<T> RepeatImpl<T>(this IEnumerable<T> sequence, int count)
+        /// <summary>
+        /// Repeats the specific sequences indefinitely, or equivalently, creates a circular sequence
+        /// from the original source sequence.
```

Comment:
I'm not sure if the clarification (everything after "or equivalently") is adding much value. If "indefinitely" may be not clear enough, I suggest replacing with "forever". Otherwise just leave it at "Repeats the specific sequences indefinitely"